sigopt/cli/commands/experiment/unarchive.py

```diff
@@ -0,0 +1,17 @@
+import click
+from sigopt.factory import SigOptFactory
+from ...arguments import project_option
+from ..base import unarchive_command
+
+
+@unarchive_command.command("experiment")
+@project_option
+@click.argument("EXPERIMENT_IDS", nargs=-1)
```

Comment:
can you add some validation to this input? IDs are always digits

Reply:
Done.